.github/workflows/integration-test.yml

```
@@ -348,6 +366,7 @@ jobs:
       - test-integration-loki
       - test-integration-pulsar
       - test-integration-splunk
+      - test-integration-redis
```

```suggestion
      - test-integration-redis
      - test-integration-splunk
```

(To keep the alphabetical sorting.)